js/classes/entities/midground/weapon/slingshot.js

```diff
@@ -115,6 +123,11 @@ class Slingshot {
         return 0.25;
     }
 
+    /** The time it takes to reload the slingshot in seconds */
```

Comment:
Are we going to add Reload time back in? It's just a comment here. Good idea.

Reply:
I'll eventually add Reload time back in. The reason I commented it out was due to all the bugs it introduced.